packages/account/src/Components/poi/status/unsupported/unsupported.jsx

```diff
@@ -20,11 +21,16 @@ const checkNimcStep = documents => {
 const Unsupported = ({ country_code, ...props }) => {
     const [detail, setDetail] = React.useState(null);
     const toggleDetail = index => setDetail(index);
+
     const documents = getDocumentIndex({
         setDetail,
         country_code,
     });
 
+    if (props?.manual && props.manual.status === 'pending') {
+        return <UploadComplete is_from_external />;
```

Comment:
What's the reason you're passing `is_from_external` here? Isn't it supposed to be passed only from CFD MT5 modal?

Reply:
Will check this